server/src/routes/routes.js

```diff
@@ -1,17 +1,21 @@
 import express from 'express';
 import signupValidator from '../validators/signupValidator';
 import loginValidator from '../validators/loginValidator';
-import userFieldsErrors from '../middleware/userFieldsErrors';
+import fieldsErrors from '../middleware/fieldsErrors';
 import userController from '../controllers/userController';
 import auth from '../middleware/auth';
```

Comment:
call it 'requireAuthentication' or something so its more clear what it is doing.